.pre-commit-config.yaml

```diff
@@ -83,6 +84,7 @@ repos:
       entry: 'Utilities/Hooks/kw-pre-commit'
       language: system
       stages: [pre-commit]
+      pass_filenames: false
```

Comment:
Not clear to me why the `pass_filenames`  value is changed from its default of `true`. Is this hook run on all file names? That would be a bit strange. My expectation is that it is run just on the changed+added files, so no need to change the `pass_filenames` as the default is appropriate. What am I missing?